### ISSUE TITLE
Link should be accessible in terminal accessibility help

### ISSUE THREAD

**Issue by lramos15** [microsoft/vscode#172262]
Testing #171862

It says to read more information at a link in the dialog, but shouldn't keyboard users be able to tab and hit enter on that link to open it? I'm trying to think how a keyboard user would open that link. I also cannot seem to copy it with my mouse

**Comment by meganrogge** [comment]
Maybe change role to document - check what screen readers use to open links and go with that

**Comment by meganrogge** [comment]
To verify on Mac:

- Enable VoiceOver
- Use `VO key (found in VoiceOver settings) + Cmd + L` to go to the link and hit enter or space to open it
- Use `VO key + Ctrl + Option/Alt + RightArrow/LeftArrow` to navigate through the elements
